hazenlib/spatial_resolution.py

```diff
@@ -461,7 +461,7 @@ def calculate_mtf_for_edge(dicom, edge, report_path=False):
         axes[9].set_title('line spread function')
         axes[9].plot(lsf)
         axes[10].set_title('normalised MTF')
-        axes[10].plot(norm_mtf)
+        axes[10].plot(norm_mtf[60:120])
```

Comment:
I think the modulation transfer function (MTF) we want to see is the first half of this array! Could you also make this more generalised? so rather than being `[0:60]` it could be something like `[0:(0.5*len(norm_mtf))]`.